x-pack/plugins/canvas/server/lib/create_handlers.js

```diff
@@ -20,8 +20,20 @@ export const createHandlers = (request, server) => {
     elasticsearchClient: async (...args) => {
       // check if the session is valid because continuing to use it
       if (server.plugins.security) {
-        const authenticationResult = await server.plugins.security.authenticate(request);
-        if (!authenticationResult.succeeded()) throw boom.unauthorized(authenticationResult.error);
+        try {
+          const authenticationResult = await server.plugins.security.authenticate(request);
+          if (!authenticationResult.succeeded())
```

Comment:
Surround with `{ }` to stay compliant with code standards.

Reply:
![screenshot 2018-10-26 10 08 37](https://user-images.githubusercontent.com/404731/47581732-272c1900-d907-11e8-81ed-59f62506a100.png)
Prettier handles all this kind of stuff.

Reply:
Interesting... then `eslint` is not in sync with our style guide.  Besides, `eslint` should not be doing this... this should be handled by `prettier`.

https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#always-use-braces-for-conditionals-and-loops